cypress/integration/hw_15.js

```diff
@@ -0,0 +1,426 @@
+    cy.get(DATA_LOCATORS.userAge).type(data.RANDOM_AGE)
+    .should("have.value", data.RANDOM_AGE);
+});
+    
+it('Filling data in the "Salary" field and verification', () => {
+    cy.get(DATA_LOCATORS.salary).type(data.RANDOM_NUM)
+    .should("have.value", data.RANDOM_NUM);
+});
+    
+it('Filling data in the "Department" field and verification', () => {
+    cy.get(DATA_LOCATORS.department).type(data.RANDOM_ADDRESS)
+    .should("have.value", data.RANDOM_ADDRESS);
+});
+
+it('Submitting data', () => {
+    cy.get(DATA_LOCATORS.submit).click()
+});
+
+it ('Clicking on searchbox field and search user by First Name', () => {
+    cy.get(DATA_LOCATORS.searchBox).type(data.RANDOM_NAME)
+    .should("have.value", data.RANDOM_NAME);
+});
+
+it ('Deleting First Name value from searchbox', () => {
+    cy.get(DATA_LOCATORS.searchBox).click()
+    .clear()
+});
+
+it ('Clicking on searchbox field and search user by LAST Name', () => {
+    cy.get(DATA_LOCATORS.searchBox).type(data.RANDOM_LAST_NAME)
+    .should("have.value", data.RANDOM_LAST_NAME);
+});
+
+it ('Deleting Last Name value from searchbox', () => {
+    cy.get(DATA_LOCATORS.searchBox).click()
+    .clear()
+});
+
+it ('Clicking on searchbox field and search user by E-Mail', () => {
+    cy.get(DATA_LOCATORS.searchBox).type(data.RANDOM_EMAIL)
+    .should("have.value", data.RANDOM_EMAIL);
+});
+
+it ('Deleting E-mail value from searchbox', () => {
+    cy.get(DATA_LOCATORS.searchBox).click()
+    .clear()
+});
+
+it ('Clicking on searchbox field and search user by Age', () => {
+    cy.get(DATA_LOCATORS.searchBox).type(data.RANDOM_AGE)
+    .should("have.value", data.RANDOM_AGE);
+});
+
+it ('Deleting Age value from searchbox', () => {
+    cy.get(DATA_LOCATORS.searchBox).click()
+    .clear()
+});
+
+it ('Clicking on searchbox field and search user by Salary', () => {
+    cy.get(DATA_LOCATORS.searchBox).type(data.RANDOM_NUM)
+    .should("have.value", data.RANDOM_NUM);
+});
+
+it ('Deleting Salary value from searchbox', () => {
+    cy.get(DATA_LOCATORS.searchBox).click()
+    .clear()
+});
+
+it ('Clicking on searchbox field and search user by Department', () => {
+    cy.get(DATA_LOCATORS.searchBox).type(data.RANDOM_ADDRESS)
+    .should("have.value", data.RANDOM_ADDRESS);
+});
+
+it ('Deleting Department value from searchbox', () => {
+    cy.get(DATA_LOCATORS.searchBox).click()
+    .clear()
+});
+});
+
+describe('Sorting table', () => {
+    it('Sort First Name by desc and verification', () => {
+      cy.get(DATA_LOCATORS.firstNameColumn) 
+      .within(() => {
+        cy.get(DATA_LOCATORS.generalColumnHeader)
+        .should('have.length', 0) 
+  
+        cy.log('**sort by desc**')
+        cy.contains(DATA_LOCATORS.columnHeader, 'First Name').click()
+        .then(func.toStrings)
+        .then(func.toNumbers)
+        .then((names) => {
+            const sorted = _.sortBy(names)
+            expect(names).to.deep.equal(sorted)
+        })
+      });
+    });
+
```

Comment:
compare style between fist selected it and second. Apply formatting into your spec